velocity/tool/src/java/org/sakaiproject/cheftool/menu/MenuImpl.java

```diff
@@ -46,7 +46,7 @@
 	protected boolean m_enabled = true;
 
 	/** The MenuItems, in order. */
-	protected List m_items = new Vector();
+	protected List<MenuItem> m_items = new Vector<MenuItem>();
```

Comment:
You can use the diamond operator here: `new Vector<>()`, or ideally: `new ArrayList<>()`